### PR TITLE
fix: filter __arana prefix table name

### LINKED ISSUE #393: [Bug] `SHOW TABLE STATUS FROM employees;` show arana core table
**What happened**:
image

**What you expected to happen**:
Same result as `show tables`

**How to reproduce it (as minimally and precisely as possible)**:
```sql
SHOW TABLE STATUS FROM employees;
```

**Anything else we need to know?**:
Provide filter, filter out uniformly arana core table

### PR DESCRIPTION
**Pull request by robbinhan**
**What this PR does**:

filter __arana prefix table name 

**Which issue(s) this PR fixes**:

Fixes #393

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by baerwang** [comment]
use Provide filter, filter out uniformly arana core table

File: pkg/runtime/plan/dal/show_table_status.go [hunk 1]
```diff
@@ -112,6 +113,18 @@ func (s *ShowTableStatusPlan) ExecIn(ctx context.Context, conn proto.VConn) (pro
...
+		switch val := next.(type) {
+		case mysql.TextRow, mysql.BinaryRow:
+			return true
+		case rows.VirtualRow:
+			vr = val
+		default:
+			return true
+		}
+		tableName := vr.Values()[0].(string)
+		return strings.HasPrefix(tableName, aranaSystemTablePrefix)
```
Comment (dongzl) [inline pkg/runtime/plan/dal/show_table_status.go:127]: `aranaSystemTablePrefix`  is  a constant at the `show_tables` file, suggest define it to a common file.

**Review by dongzl — CHANGES_REQUESTED** [review]

**Comment by dongzl** [comment]
> 

what is the `Provide filter`?

**Comment by dongzl** [comment]
Close by #431

**Pull request closed without merging**